loony.nim

```diff
@@ -215,8 +215,9 @@ proc pushImpl[T](queue: LoonyQueue[T], el: T,
   # Ensure all writes in STOREBUFFER are committed. By far the most costly
   # primitive; it will be preferred while proving safety before working towards
   # optimisation by atomic reads/writes of cache lines related to el
-  when forcedCoherance:
-    atomicThreadFence(ATOMIC_RELEASE)
+  # when forcedCoherance:
+  #   atomicThreadFence(ATOMIC_RELEASE)
+  atomicThreadFence(ATOMIC_RELEASE)
```

Comment:
May as well remove it; if you leave it, it's useful as documentation: _we don't do this anymore because..._